library/objective-c/BUILD

```diff
@@ -8,12 +8,14 @@ exports_files([
 objc_library(
     name = "envoy_engine_objc_lib",
     srcs = [
-        "EnvoyEngine.m",
+        "EnvoyEngineImpl.h",
```

Comment:
Shouldn't this be in `hdrs`?

Reply:
Only if it's meant to be used externally - i.e., I don't think it'd be something we'd want to expose as part of the framework's public interface. Not entirely sure of the interplay with the Swift layer though. If it needs to be surfaced to `hdrs` in order for Swift to access it, then we can move it.